src/current/v25.2/known-limitations.md

```diff
@@ -26,6 +26,10 @@ This section describes newly identified limitations in CockroachDB {{ page.versi
 
 {% include {{ page.version.version }}/known-limitations/rls-values-on-conflict-do-nothing.md %}
 
+#### Row-level security metadata leakage
+
+{% include {{ page.version.version }}/known-limitations/rls-visibility-issue.md %}
```

Comment:
Actually one more suggestion -- this structure in Known Limitations creates two duplicate `Security and privileges` headings. I would recommend grouping both KL includes into bullets under a single `Row-level security` heading to keep this cleaner.